konfera/migrations/0010_auto_20161117_0652.py

```diff
@@ -0,0 +1,44 @@
+                ('company_id', models.CharField(max_length=32)),
+                ('tax_id', models.CharField(max_length=32)),
+                ('vat_id', models.CharField(max_length=32)),
+                ('about_us', models.TextField(blank=True)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.AddField(
+            model_name='event',
+            name='organizer',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='organized_events', to='konfera.Organizer'),
```

Comment:
`SET_NULL` would be better choice than `CASCADE`.

@ricco386 I think we should check every foreign key in the whole project and make sure `on_delete` works as expected.